app/models/basic_project.rb

```diff
@@ -0,0 +1,43 @@
+# == Schema Information
+#
+# Table name: basic_projects
+#
+#  created_at         :datetime         not null
+#  division_id        :integer
```

Comment:
It is nullable on other models (but validated as present). Maybe there should be a new issue to change them all to not null.